packages/rehype-parse/types/index.d.ts

```diff
@@ -1,13 +1,14 @@
 // TypeScript Version: 3.5
 import {Parser, Plugin} from 'unified'
 import {Node, Parent} from 'unist'
+import {HastUtilFromParse5Options} from 'hast-util-from-parse5'
 
 declare namespace rehypeParse {
   interface Parse extends Plugin<[RehypeParserOptions?]> {
     Parser: Parser
   }
 
-  interface RehypeParserOptions {
+  interface RehypeParserOptions extends HastUtilFromParse5Options {
```

Comment:
`HastUtilFromParse5Options` includes `file`
https://github.com/syntax-tree/hast-util-from-parse5/blob/d43f3ef08f713d1b15e0cbc1dad54112d241c2a8/types/index.d.ts#L21-L28

Which isn't doesn't appear to be an option available to `RehypeParserOptions`
https://github.com/rehypejs/rehype/tree/master/packages/rehype-parse#options

Might we want to `Omit` it? or `Pick` the `space` and `verbose` attributes specifically?

Reply:
I chose `Pick`, because in code these properties are explicitly picked as well. As a result, if new options would be added to `hast-util-from-parse5`, these types would still be valid.